snuba/settings/__init__.py

```diff
@@ -208,6 +208,9 @@
 # Enable replays ingestion
 ENABLE_REPLAYS_CONSUMER = os.environ.get("ENABLE_REPLAYS_CONSUMER", False)
 
+# Used to tag consumers as experimental to avoid them crash looping in production
+EXPERIMENTAL_CONSUMER_GROUPS: Set[str] = set()
```

Comment:
Why a dedicated setting? We already have experimental datasets, I would say by default, all storages and consumers of an experimental dataset should be experimental.

In the same way you should not have a consumer marked as experimental for a dataset that is marked as production. Can we use the same setting ?